backend/api/src/test/kotlin/software/shonk/application/service/ShorkServiceTest.kt

```diff
@@ -346,4 +346,37 @@ class ShorkServiceTest {
         assertEquals(Winner.DRAW, result.result.winner)
         assertTrue(result.visualizationData.isEmpty())
     }
+
+    @Test
+    fun `test get lobby settings for a valid lobby`() {
+        val shorkService = ShorkService(Shork())
+        val lobbyId = shorkService.createLobby("playerA").getOrThrow()
+        shorkService.setLobbySettings(lobbyId, Settings())
+        val result = shorkService.getLobbySettings(lobbyId)
+        val defaultSettings = Settings()
+        val testSettings = result.getOrThrow()
+
+        assertTrue(
+            testSettings ==
+                InterpreterSettings(
+                    coreSize = defaultSettings.coreSize,
+                    instructionLimit = defaultSettings.instructionLimit,
+                    initialInstruction = defaultSettings.initialInstruction,
+                    maximumTicks = defaultSettings.maximumTicks,
+                    maximumProcessesPerPlayer = defaultSettings.maximumProcessesPerPlayer,
+                    readDistance = defaultSettings.readDistance,
+                    writeDistance = defaultSettings.writeDistance,
+                    minimumSeparation = defaultSettings.minimumSeparation,
+                    separation = defaultSettings.separation,
+                    randomSeparation = defaultSettings.randomSeparation,
+                )
+        )
+    }
```

Comment:
It would be even nicer if you re-used the same settings instead of creating the default settings twice